JS/app.js

```diff
@@ -22,87 +24,135 @@ function AdImage(name, image){
-        if(imageClicked === 'picLeft'){
-        // do logic to increment the number
-        AdImage.allImages[leftIndex].clicked++;
-        // console.log('left pic selected');
-
-        } else if(imageClicked === 'picRight'){
-        AdImage.allImages[rightIndex].clicked++;
-        // console.log('right pic selected');
-        } else if(imageClicked === 'picCenter'){
-            AdImage.allImages[centerIndex].clicked++;
-            // console.log('center pic selected');
-            }
+        // if(imageClicked === 'picLeft'){
+        // allImages[leftIndex].clicked++;
 
-    } else{
-        alert('You didn\'t select an image. ');
-    } 
+        // } else if(imageClicked === 'picRight'){
+        // allImages[rightIndex].clicked++;
+        
+        // } else if(imageClicked === 'picCenter'){
+        //     allImages[centerIndex].clicked++;
+        //     }
+
+        for(var i = 0; i < AdImage.allProducts.length; i++){
+            if(imageClicked === AdImage.allProducts[i].name){
+                AdImage.allProducts[i].clicked++;
+                console.log(AdImage.allProducts[i].clicked);
+            }
+        }
+        renderAdImages();
+    }
     
     // currentPic = [leftIndex, rightIndex, centerIndex];
-    // console.log(currentPic);
     
-    if(imageVote === 25){
-            pictureParent.removeEventListener('click', handleClickOnImage);
-            //console.log("you completed the voting")
+    // if(imageVote === 25){
+    //         pictureParent.removeEventListener('click', handleClickOnImage);
             // output to the browser the results
             
-            for(var i=0; i < AdImage.allImages.length; i++) {
-            var pix = AdImage.allImages[i];
+            // for(var i=0; i < AdImage.allImages.length; i++) {
+            // var pix = AdImage.allImages[i];
             // console.log(`${pix.name} received ${pix.clicked} votes with ${pix.views} views`);
             // var node = document.createElement("li");                 // Create a <li> node ---taken from w3 schools
             // var textnode = document.createTextNode(`${pix.name} received ${pix.clicked} votes with ${pix.views} views`);         // Create a text node
             // node.appendChild(textnode);                              // Append the text to <li>
             // document.getElementById("outputList").appendChild(node);     // Append <li> to <ul> with id="myList"
-            }
-                } else{
-                    renderAdImages();
-                }
-    }
+            // }
+            //     } else{
+            //         renderAdImages();
+            //     }
+    
 //closing comment - Create-picture-clicking function. Must be defined before our event listener
 
 function renderAdImages(){
-  do{
+//   do{
     leftIndex = randomPic();
     rightIndex = randomPic();
     centerIndex = randomPic();
 
-    } while(leftIndex === rightIndex || leftIndex === centerIndex || rightIndex === centerIndex){
```

Comment:
Currently, your code is not preventing the same image from rendering multiple time. I see you got rid of this conditional logic which was preventing that from happening?